docs/GraFx-Studio/guides/template-variables/define/index.md

```diff
@@ -83,20 +100,44 @@ A placeholder that can hold "True" or "False".
 
 A Placeholder to hold a numeric value. Different than a text variable, you can set specific elements related to numbers.
 
-## Default state
+## Variable settings
+
+### Variable name
+
+![screenshot](var07.png)
+
+In the list of variable, you can double click or "Rename" the variable Name.
+
+!!! important "Variable name vs Display Label"
+
+    The variable name is the technical name, that will be used for further automation.
+    In the general properties you can set the display label, that will be shown in the [Studio UI interface](/GraFx-Studio/concepts/template-management/?h=studio#studio-ui).
+
+### Default state
+
+![screenshot](var04.png)
 
 The default value your template variable will have.
 
-## Basic properties:
+### General properties:
+
+![screenshot](var05.png)
 
-- Name: Cannot be empty, must be unique
 - Type: See [types](#types) above
 - [Visible](#visibility-conditions): Enabled by default
-- Read-only: Disabled by default
+- Required: Disabled by default
+
+Below, there will be type-specific settings
```

Comment:
Private data is mentioned below, but that is not type-specific. It's available for all variable types.